yt_dlp/extractor/abc.py

```diff
@@ -299,47 +329,60 @@ def tokenize_url(url, token):
+        # hls-latest: same as hls, but sometimes has separate audio tracks
+        # Note: pre-merged formats in hls-latest are treated as video-only if audio-only tracks
+        # are present, so we extract both types
+        for label in ('hls', 'hls-latest'):
+            for sd_url in traverse_obj(stream, ('streams', label, ('1080', '720', 'sd', 'sd-low'), {str})):
+                fmts = self._extract_m3u8_formats(
+                    tokenize_url(sd_url, token), video_id, 'mp4',
+                    entry_protocol='m3u8_native', m3u8_id=label, fatal=False)
+                if fmts:
+                    formats.extend(fmts)
+                    break
 
         subtitles = {}
-        src_vtt = stream.get('captions', {}).get('src-vtt')
+        src_vtt = traverse_obj(stream, ('captions', 'src-vtt'))
         if src_vtt:
             subtitles['en'] = [{
                 'url': src_vtt,
                 'ext': 'vtt',
             }]
 
-        is_live = video_params.get('livestream') == '1'
+        title = unescapeHTML(traverse_obj(video_params, 'title', 'seriesTitle', 'showTitle'))
 
         return {
             'id': video_id,
             'title': title,
-            'description': video_params.get('description'),
-            'thumbnail': video_params.get('thumbnail'),
-            'duration': int_or_none(video_params.get('eventDuration')),
-            'timestamp': parse_iso8601(video_params.get('pubDate'), ' '),
-            'series': unescapeHTML(video_params.get('seriesTitle')),
-            'series_id': video_params.get('seriesHouseNumber') or video_id[:7],
+            'series_id': traverse_obj(
+                video_params,
+                ('analytics', 'oztam', 'seriesId'), 'seriesHouseNumber') or video_id[:7],
             'season_number': int_or_none(self._search_regex(
                 r'\bSeries\s+(\d+)\b', title, 'season number', default=None)),
             'episode_number': int_or_none(self._search_regex(
-                r'\bEp\s+(\d+)\b', title, 'episode number', default=None)),
+                r'\bEp(?:isode)?\s+(\d+)\b', title, 'episode number', default=None)),
             'episode_id': house_number,
             'episode': self._search_regex(
-                r'^(?:Series\s+\d+)?\s*(?:Ep\s+\d+)?\s*(.*)$', title, 'episode', default='') or None,
-            'uploader_id': video_params.get('channel'),
+                r'^(?:Series\s+\d+)?\s*(?:Ep(?:isode)?\s+\d+)?\s*(.*)$', title, 'episode', default='') or None,
+            'is_live': video_params.get('livestream') == '1' or stream['type'] == 'livestream',
             'formats': formats,
             'subtitles': subtitles,
-            'is_live': is_live,
+            **traverse_obj(video_params, {
+                'description': ('description', {str}),
+                'thumbnail': ((
+                    ('images', lambda _, v: v['name'] == 'episodeThumbnail', 'url'), 'thumbnail'), {str}, any),
+                'chapters': ('cuePoints', ..., {
```

Comment:
a chapter without a start time could cause problems so let's filter here
```suggestion
                'chapters': ('cuePoints', lambda _, v: int(v['start']) is not None, {
```